src/components/Card/Card.test.tsx

```diff
@@ -2,28 +2,26 @@ import React from "react";
+        description="Fynd is your one stop destination for buying, selling & payments,
+  Logistics, Order fulfillment, News, Games, Quizzes & a lot
+  more."
+        image={<img src="static/media/src/components/Card/card1.jpeg" />}
```

Comment:
Inspite of naming card1, let's add a meaningful name like card-{variant}.jpeg

Reply:
Will be adding more testcases in a separate PR